doc/whatsnew/fragments/7578.bugfix

```diff
@@ -0,0 +1,3 @@
+Handle patterns in ignored-modules when checking for no-name-in-module
```

Comment:
```suggestion
The ``ignored-modules`` option will now be correctly taken into account for ``no-name-in-module``.
```

Reply:
The "important" part is the dot, to make pre-commit pass, but I also suggested a clearer summary, let me know what you think.

Reply:
Thanks!